packages/api/src/hooks/useActiveWalletAccountVerbose.ts

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
+import useActiveWalletAccount from './useActiveWalletAccount';
+import useMT5AccountsList from './useMT5AccountsList';
+import useWalletAccountsList from './useWalletAccountsList';
+import useDxtradeAccountsList from './useDxtradeAccountsList';
+
+/** A custom hook that returns the wallet object for the current active wallet, containing `transfer_options` list. */
+const useActiveWalletAccountVerbose = () => {
```

Comment:
Hey man maybe we can create something like `useActiveLinkedToTradingAccount` 👀

Reply:
After checking with @heorhi-deriv I think I'm gonna change the currently unused `useAllAvailableAccounts` hook to do the part of the logic of the `useActiveWalletAccountVerbose`, then we don't need `useActiveWalletAccountVerbose`

Reply:
This hook no longer exists, have created `useAllAccountsList`